src/wagtail_vector_index/ai_utils/backends/llm.py

```diff
@@ -110,3 +123,6 @@ def _get_llm_embedding_model(self) -> llm.EmbeddingModel:
     def embed(self, inputs: Iterable[str]) -> Iterator[list[float]]:
         model = self._get_llm_embedding_model()
         yield from model.embed_multi(inputs)
+
+    async def aembed(self, inputs: Iterable[str]) -> Iterator[list[float]]:
+        raise NotImplementedError("Async embed is not supported by this backend.")
```

Comment:
Is there a reason why we're not making this the default implementation (on `BaseEmbeddingBackend`)?